test/plots/us-presidential-forecast-2016.js

```diff
@@ -0,0 +1,22 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export default async function() {
+  const data = await d3.csv("data/us-presidential-forecast-2016-histogram.csv", d3.autoType);
+  return Plot.plot({
+    x: {
+      label: "Electoral votes for Hillary Clinton →",
+      labelAnchor: "left",
+      ticks: [100, 200, 300, 400, 500]
+    },
+    y: {
+      ticks: 5,
+      tickFormat: "%"
+    },
+    marks: [
+      Plot.barY(data, {x: "dem_ev", y: "prob", shapeRendering: "crispEdges", fill: "dem_ev", stroke: "dem_ev" }),
```

Comment:
We don’t want a bar here because the _x_-scale is quantitative (number of electoral votes), not ordinal. I’ll rewrite this example.